Updating title tags and meta descriptions (#2509)
pulumi/pulumi-hugo: themes/default/content/learn/embedding-pulumi/_index.md

---
title_tag: Embedding Pulumi Overview | Learn Pulumi
title: "Embedding Pulumi"
layout: module
date: 2021-12-15
draft: false
description: |
    Explore building a custom infrastructure API to embed Pulumi in other
    contexts.

> meta_desc: In this tutorial, we will explore how you can use Pulumi as an API for your infrastructure deployment and maintenance.

Reviewer: ```suggestion
meta_desc: In this tutorial, we will explore how to use Pulumi to build an infrastructure API for deployment and maintenance.
```
